UKHomeOffice/sgmr-service
# SPCR-400 Country/Nationality selector

### AC / Description of changes
- update person nationality to use the country/nationality autocomplete component
- update person document issuing state to use the country/nationality autocomplete component
- update person data to submit to pass only the 3 digit ISO codes for both these fields

### To Test

- create a person
- start typing in the document issuing state field - it should show a selection of countries
- start typing in the nationality field - it should show a selection of countries
- select your country for each and submit user
- it should submit without error


### Notes

If you return to edit a person you will see the ISO codes not the country names for nationality. This will be fixed at a later date to improve UX

---

**Remember to merge to feature branches not `master`.**
